docs/Course Wiki/MATH Course Offerings/MATH285.md

```diff
@@ -0,0 +1,54 @@
+# MATH285 
+
+MATH285 (Intro Differential Equations) is a 3-credit hour course that is required for ECE majors as a part of the Foundational Mathematics and Science requirement. It is also required for many other engineering majors. It is offered in the fall, spring, and summer semesters.
+
+## Content Covered
+
+- First order ODEs
+- Second-order ODE's
+- Damped/Undamped/Forced/Unforced Harmonic Oscillators
+- Higher order ODEs
+- Boundary Value Problems
+- Fourier Series / Periodically Forced Oscillators
+- Heat Equation
+- Wave Equation
+- Laplace's Equation
+- Sturm-Liouville Theory and Eigenfunction Series
+
+MATH285 is a foundational course in many concepts crucial for any ECE major. Its impact is most directly seen in [ECE210](ECE210.md) and courses that depend on ECE210.
+
+## Prerequisites
+
+- [MATH241](MATH241.md)
+  
+The official prerequisite for MATH285 is MATH241, which provides a background in multivariable calculus. Since MATH285 goes into partial differential equations and 2-D heat equations
+towards the end, this knowledge is very helpful.
+
+## When to Take It
+
+ECE majors typically take MATH285 in their freshman year spring semester or during their sophomore year, depending on how much math credit they come in with. MATH285 is a vital pre/corequisite for [ECE210](ECE210.md), so taking this course as soon as possible is highly recommended, especially for EE majors. 
+
+## Course Structure
+
+This class meets three times a week for a 50-minute Lecture-Discussion. 
```

Comment:
Can you write a bit more about this? Are there exams, quizzes in this class?  Are there homework assignments?  Any other course policies that are important to note?